new-version.sh

```diff
@@ -116,7 +116,7 @@ if [[ "${RELEASE_VERSION}" == *-rc.? ]]; then
 fi
 
 # (1) Bump python module versions
-PYTHON_MODULES=(clients/python/ integrations/airflow/ integrations/common/)
+PYTHON_MODULES=(clients/python/ integrations/airflow/ integrations/common/ integrations/dbt_bigquery/ integrations/dbt_snowflake/)
```

Comment:
You'll want to update the paths here to the new py module locations: `integrations/dbt/bigquery/`, and `integrations/dbt/snowflake/`

Reply:
Done.